.gitignore

```diff
@@ -15,4 +15,7 @@ public/app.js
 
 # Keys
 apiKeys.js
-apikeys.js
```

Comment:
For google and eventful to work on the server, eventcontroller.js requires the apikeys.js file.